Prose/Prose/ProseApp.swift

```diff
@@ -6,28 +6,11 @@
 //
 
 import App
-import SettingsFeature
 import SwiftUI
 
 @main
 struct ProseApp: App {
     var body: some Scene {
-        WindowGroup {
-            AppView()
-        }
-        .windowStyle(DefaultWindowStyle())
-        .windowToolbarStyle(UnifiedWindowToolbarStyle())
-        .commands {
-            SidebarCommands()
-
-            CommandMenu("TODO Menu") {
```

Comment:
Please remove this to re-enable the import Cmd-H shortcut. No need to have todos here.

Reply:
I don't understand, do you want me to add id or remove it? Me I didn't touch this, it's been here for ages.

Reply:
Yes, sorry, you just moved it. Let's remove it though so that we can hide windows via the system shortcut.

Reply:
Honestly, I don't think the shortcut worked anyway (I was a bit surprised actually, but since I didn't care, I didn't dig).

It's in the bar, and the action works there, but the shortcut itself does nothing.

Reply:
Ok, maybe I was blind or my French keyboard messes it up 🤷🏻‍♂️

Anyway, that's a problem for later.